### ISSUE TITLE
Unable to load package, missing src folder in dialogic-svelte.

### ISSUE THREAD

**Issue by csell5** [ArthurClemens/dialogic#10]
I was trying to give the package a try in Svelte but it appears the npm package is missing the 'src' folder or miss-configured and not trying to use the dist folder.

npm install dialogic-svelte --save-dev

```
[!] Error: Could not load [...]/node_modules/dialogic-svelte/src/index.js (imported by src/App.svelte): ENOENT: no such file or directory, open '[...]/node_modules/dialogic-svelte/src/index.js'
```

The error is correct, the src folder isn't there.

**Comment by ArthurClemens** [comment]
Thanks for reporting. Fixed in `0.10.3`.

**Comment by csell5** [comment]
Awesome, thank you.. I've also verified it works as expected.
